src/zc/zodbdgc/README.test

```diff
@@ -100,10 +102,14 @@ cross db loops
 No garbage yet, because everything's reachable.
 
     >>> from ZODB.utils import u64, p64
-    >>> print u64(conn1.root.a._p_oid), u64(conn1.root.b._p_oid)
-    2204 2205
-    >>> print u64(conn2.root.x._p_oid), u64(conn3.root.x._p_oid)
-    1 2
+    >>> print( u64(conn1.root.a._p_oid) )
+    2204
+    >>> print( u64(conn1.root.b._p_oid) )
+    2205
+    >>> print( u64(conn2.root.x._p_oid) )
+    1
+    >>> print( u64(conn3.root.x._p_oid) )
+    2
```

Comment:
Simple mechanical transform. I'll clean it up.

Presumably? I didn't want to mess with it anymore than needed...I personally find doctests difficult to debug.